assets/xml/objects/object_stk2.xml

```diff
@@ -1,86 +1,105 @@
-        <Texture Name="object_stk2_Tex_007E48" OutName="tex_007E48" Format="i8" Width="32" Height="64" Offset="0x7E48" />
-        <TextureAnimation Name="object_stk2_Matanimheader_008658" Offset="0x8658" />
+        <Animation Name="gSkullKidHorseRideLoopAnim" Offset="0x49C8" /> <!-- Original name is "lost_horse_wait" -->
+        <Animation Name="gSkullKidGlitchyHorseRideStartAnim" Offset="0x70DC" /> <!-- Original name is "lost_house" -->
+
+        <!-- Assets for the curse effect overlayed on top of Majora's Mask when Skull Kid curses Link -->
+        <DList Name="gSkullKidMajorasMaskCurseOverlayDL" Offset="0x7840" />
+        <Texture Name="gSkullKidMajorasMaskCurseOverlayMaskTex" OutName="skull_kid_majoras_mask_curse_overlay_mask" Format="i8" Width="32" Height="32" Offset="0x7A48" />
+        <Texture Name="gSkullKidMajorasMaskCurseOverlayElectricityTex" OutName="skull_kid_majoras_mask_curse_overlay_electricity" Format="i8" Width="32" Height="64" Offset="0x7E48" />
+        <TextureAnimation Name="gSkullKidMajorasMaskCurseOverlayTexScroll" Offset="0x8658" />
+
+        <!-- Assets for the effects displayed on the sides of the screen when Skull Kid curses Link or calls down the moon -->
         <DList Name="object_stk2_DL_008920" Offset="0x8920" />
         <DList Name="object_stk2_DL_008A38" Offset="0x8A38" />
         <Texture Name="object_stk2_Tex_008B50" OutName="tex_008B50" Format="i8" Width="32" Height="64" Offset="0x8B50" />
         <Texture Name="object_stk2_Tex_009350" OutName="tex_009350" Format="i8" Width="32" Height="32" Offset="0x9350" />
         <Texture Name="object_stk2_Tex_009750" OutName="tex_009750" Format="i8" Width="32" Height="64" Offset="0x9750" />
         <TextureAnimation Name="object_stk2_Matanimheader_009F60" Offset="0x9F60" />
+
+        <!-- Clock Tower Ocarina Animations -->
         <Animation Name="object_stk2_Anim_00A104" Offset="0xA104" />
         <Animation Name="object_stk2_Anim_00A2BC" Offset="0xA2BC" />
         <Animation Name="object_stk2_Anim_00A4C4" Offset="0xA4C4" />
         <Animation Name="object_stk2_Anim_00A6D4" Offset="0xA6D4" />
+
+        <!-- Clock Tower Ocarina Limb DisplayLists -->
         <DList Name="object_stk2_DL_00A840" Offset="0xA840" />
+
+        <!-- Clock Tower Ocarina Textures -->
         <Texture Name="object_stk2_Tex_00A958" OutName="tex_00A958" Format="rgba16" Width="32" Height="16" Offset="0xA958" />
-        <!-- <Blob Name="object_stk2_Blob_00AD58" Size="0x28" Offset="0xAD58" /> -->
-        <Animation Name="object_stk2_Anim_00C270" Offset="0xC270" />
+
+        <!-- Clock Tower Ocarina Limbs -->
+        <Limb Name="object_stk2_Standardlimb_00AD58" Type="Standard" EnumName="OBJECT_STK2_LIMB_01" Offset="0xAD58" />
+
+        <!-- Clock Tower Ocarina Skeleton -->
+        <Skeleton Name="object_stk2_Skel_013158" Type="Flex" LimbType="Standard" LimbNone="OBJECT_STK2_LIMB_NONE" LimbMax="OBJECT_STK2_LIMB_MAX" EnumName="object_stk2_Limbs" Offset="0xAD68" />
```

Comment:
I was going to wait until I decomp'd whatever actor uses these (can't remember which it is), but yeah, I might as well name stuff if I know what it is. I'll leave the animations unnamed though; they can't be viewed in Z64Utils, since it can't find the skeleton (I presume because it has only one limb). When I decompile the actor, I'll hopefully have more context to name the animations, though I also have the names from MM3D if push comes to shove.

Reply:
Given what the drum does, it probably just tumbles or falls over or something.

Reply:
From MM3D, it looks like there's an animation for falling out of his hand, one for hitting the ground, and two for when he's juggling it. The problem is that since I can't view the 64 animations, I can't map them to the 3D animations. Now the animations are *usually* in alphabetical order, so I could simply guess, but I'd rather wait for more context and be correct than guess now and be wrong.

Reply:
The blob is a TexAnim, I think? Not completely sure.

Reply:
Which blob, 0xAD58? That's the the single limb of the OoT skeleton.

Reply:
Oh, the combined view confused me 😕

Reply:
Yeah, the only blobs left *should* be those weird 0x00, 0x11 blobs I posted about in the other comment. All other ones should be taken care of.